datapipelines/src/test/java/io/customer/sdk/SdkInitializationLoggerTest.kt

```diff
@@ -0,0 +1,111 @@
+package io.customer.sdk
+
+import io.customer.commontest.extensions.assertCalledOnce
+import io.customer.datapipelines.testutils.core.JUnitTest
+import io.customer.sdk.core.module.CustomerIOModule
+import io.customer.sdk.core.module.CustomerIOModuleConfig
+import io.customer.sdk.core.util.Logger
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.runs
+import io.mockk.slot
+import org.amshove.kluent.internal.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class SdkInitializationLoggerTest : JUnitTest() {
```

Comment:
`InitLoggerTest` might be a more appropiate name as we are unit testing the InitLogger